### PR TITLE
[PEPPER-483] Update LMS Germline Consent Invitation Emails

### PR DESCRIPTION
**Pull request by rfricke-asymmetrik**
## Context
[PEPPER-483](https://broadworkbench.atlassian.net/browse/PEPPER-483)

_Say a few words about the bigger picture beyond this PR. How do your changes improve life for our users? Link directly to the jira ticket (just type "DDP-..." and it'll automatically link) to help your teammates quickly get up to speed on the motivation for these changes._

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [x] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

[DDP-483]: https://broadinstitute.atlassian.net/browse/DDP-483?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PEPPER-483]: https://broadworkbench.atlassian.net/browse/PEPPER-483?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
     </tr>
     <tr style="width: 540px;">
         <td style="padding: 50px 0 0 0;">
-            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.participant.firstName-,</p>
+            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear Participant,</p>
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html]: I believe this is incorrect. I dont think these changes are needed. We need name substituted.

File: pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html [hunk 2]
```diff
@@ -117,9 +117,8 @@
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">The LMSproject team</p>
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
                 For additional updates, follow us on
-                <a href='${social.facebook}'    target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Facebook</a>,
-                <a href='${social.instagram}'   target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Instagram</a> or
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html]: Do we have requirement to remove instagram ?

File: pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-pediatric-invitation.html [hunk 3]
```diff
@@ -82,7 +82,7 @@
     </tr>
     <tr style="width: 540px;">
         <td style="padding: 50px 0 0 0;">
-            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.proxy.firstName-,</p>
+            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear Parent,</p>
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-pediatric-invitation.html]: I dont think these changes are needed. We need name substituted.

File: pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-pediatric-invitation.html [hunk 4]
```diff
@@ -117,9 +117,8 @@
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">The LMSproject team</p>
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
                 For additional updates, follow us on
-                <a href='${social.facebook}'    target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Facebook</a>,
-                <a href='${social.instagram}'   target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Instagram</a> or
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-pediatric-invitation.html]: same as other file. Do we have requirement to remove instagram ?

**Comment by ssettipalli** [comment]
Please add/fill "deployment notes" to the ticket

File: pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html (hunk 1 above)
Comment (rfricke-asymmetrik) [reply pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html]: Maybe I misunderstood the request, but the acceptance criteria of the ticket says: 

AC2: Adult email recipients are addressed as “Participant”; blue highlighted text is “your”`

And the docs attached to the ticket have the salutation line as `Dear Participant/Parent` highlighted in blue. So I figured this meant that was meant to be changed. I can check with Whitney.

File: pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html (hunk 2 above)
Comment (rfricke-asymmetrik) [reply pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html]: I can put this back. I was looking at the docs attached to the ticket and one of the differences was the instagram was removed and twitter and facebook were swapped. But that may have been unintentional so I will check with Whitney on this as well.

File: pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html (hunk 1 above)
Comment (ssettipalli) [reply pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html]: I believe its original document for reference. Yes Whitney can confirm it.

**Review by ssettipalli — APPROVED** [review]

**Pull request merged**